ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,453 @@
+                var height = style == .horizontal ? Constants.horizontalBaseHeight : Constants.verticalBaseHeight
+                let (singleLineHeight, twoLineHeight) = labelHeight(for: traitCollection.preferredContentSizeCategory)
+
+                if twoLineTitle {
+                   height += twoLineHeight
+                } else {
+                    height += 2 * singleLineHeight
+                }
+
+                heightConstraint.constant = height
+            }
+            if #available(iOS 13, *) {
+                if previousTraitCollection.hasDifferentColorAppearance(comparedTo: traitCollection) {
+                    // Update border color
+                    switch colorStyle {
+                    case .appColor:
+                        if let window = window {
+                            layer.borderColor = UIColor(light: Colors.primaryTint30(for: window), dark: .clear).cgColor
+                        }
+                    case .neutral:
+                        layer.borderColor = Constants.defaultBorderColor.cgColor
+                    case .custom:
+                        layer.borderColor = customBorderColor.cgColor
+                    }
+                }
+            }
+        }
+    }
+
+    /// Set up the background color of the card and update the icon and text color if necessary
+    private func setupColors() {
+        switch colorStyle {
+        case .appColor:
+            primaryLabel.textColor = UIColor(light: .black, dark: Colors.gray100)
+            secondaryLabel.textColor = UIColor(light: Colors.gray600, dark: Colors.gray400)
+            iconView.tintColor = UIColor(light: Colors.gray600, dark: Colors.gray500)
+            if let window = window {
+                backgroundColor = UIColor(light: Colors.primaryTint40(for: window), dark: Colors.primaryTint30(for: window))
+                layer.borderColor = UIColor(light: Colors.primaryTint30(for: window), dark: .clear).cgColor
+            }
+        case .neutral:
+            backgroundColor = Constants.defaultBackgroundColor
+            primaryLabel.textColor = Constants.defaultTitleColor
+            secondaryLabel.textColor = Constants.defaultSubtitleColor
+            iconView.tintColor = Constants.defaultIconTintColor
+            layer.borderColor = Constants.defaultBorderColor.cgColor
+        case .custom:
+            backgroundColor = customBackgroundColor
+            primaryLabel.textColor = customTitleColor
+            secondaryLabel.textColor = customSubtitleColor
+            iconView.tintColor = customIconTintColor
+            layer.borderColor = customBorderColor.cgColor
+        }
+    }
+
+    private struct Constants {
+        static let defaultBackgroundColor = UIColor(light: .white, dark: Colors.gray900)
+        static let defaultBorderColor = UIColor(light: Colors.dividerOnPrimary, dark: .clear)
+        static let defaultTitleColor: UIColor = Colors.textPrimary
```

Comment:
do we need these 3 colors define in this private struct. defaulttitlecolor, subtitle and icontitnecolor? I know it is minor but more memory